src/popup/components/Modals/ProtocolSelect.vue

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
This is initially a `AccountCreate` component. That is adjusted to return chosen protocol.

Reply:
Maybe instead of this it would be better to reuse the `AccountCreate` modal so instead of having the account creation logic there we move it to `AccountCardAdd`.

Reply:
This is what i did.